src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2362,6 +2411,13 @@ export class StudyViewPageStore
         );
     }
 
+    @computed get filterStructVarsTableByCancerGenes(): boolean {
```

Comment:
this doesn't need to be computed because we aren't saving any computation by doing so.  i just want to make sure it's understood that making it computed does not change it's observability.